src/test/java/com/teragrep/pth_03/TicketSyntaxTests.java

```diff
@@ -303,4 +300,63 @@ void xpathTest10(String arg) throws Exception {
         // Check that 1 found
         assertEquals(1,nodesA.getLength());
     }
+
+    @ParameterizedTest
+    @ValueSource(strings = {
+            "ticket17",
+    })
+    void invalidWhereQueryTest(String arg) {
+        String fileName = "src/test/resources/antlr4/tickets/" + arg + ".txt";
+        ParserSyntaxTestingUtility parserSyntaxTestingUtility
+                = new ParserSyntaxTestingUtility(fileName, false);
+        Assertions.assertThrows(Exception.class, () -> parserSyntaxTestingUtility.syntaxParseTest(arg));
+    }
+
+    @Disabled(value = "Issue open. Can't parse second right parenthesis")
```

Comment:
Add note like `GH pth_03 #1234` so it can be easily tracked

Reply:
Very good addition, thanks. I added this to every other newly disabled test as well.